modules/vf-graphql-holochain/mutations/agent.ts

```diff
@@ -45,7 +45,7 @@ export interface OrganizationUpdateArgs {
 export type updateOrganizationHandler = (root: any, args: OrganizationUpdateArgs) => Promise<OrganizationResponse>
 
 export interface AgentCreateArgs {
-    agent: OrganizationCreateParams,
+    agent: OrganizationCreateParams & { agentType: String },
```

Comment:
```suggestion
    agent: OrganizationCreateParams & { agentType: string },
```

usually use lowercase `s` and first letters with typescript. I forget the exact nature of the difference, but I know to do this